diff --git a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib.esm/index.js b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib.esm/index.js
index 918c9fb3..73654554 100644
--- a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib.esm/index.js
+++ b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib.esm/index.js
@@ -149,6 +149,10 @@
     "arbitrum-rinkeby": { chainId: 421611, name: "arbitrum-rinkeby" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3", _defaultProvider: etcDefaultProvider("https://rpc.rollux.com", "rollux-tanenbaum") },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib/index.js b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib/index.js
index c0f624b0..dfafcb40 100644
--- a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib/index.js
+++ b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/lib/index.js
@@ -151,6 +151,10 @@
     "arbitrum-rinkeby": { chainId: 421611, name: "arbitrum-rinkeby" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3", _defaultProvider: etcDefaultProvider("https://rpc.rollux.com", "rollux-tanenbaum") },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/src.ts/index.ts b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/src.ts/index.ts
index daf97b7d..ed8dde80 100644
--- a/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/src.ts/index.ts
+++ b/node_modules/@web3-onboard/core/node_modules/@ethersproject/networks/src.ts/index.ts
@@ -189,6 +189,12 @@

     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3", _defaultProvider: etcDefaultProvider("https://rpc.rollux.com", "rollux-tanenbaum") },
 }

 /**
