diff --git a/node_modules/@ethersproject/networks/lib.esm/index.js b/node_modules/@ethersproject/networks/lib.esm/index.js
index 918c9fb3..73654554 100644
--- a/node_modules/@ethersproject/networks/lib.esm/index.js
+++ b/node_modules/@ethersproject/networks/lib.esm/index.js
@@ -168,6 +168,10 @@ const networks = {
     "arbitrum-goerli": { chainId: 421613, name: "arbitrum-goerli" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3" },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@ethersproject/networks/lib/index.js b/node_modules/@ethersproject/networks/lib/index.js
index c0f624b0..dfafcb40 100644
--- a/node_modules/@ethersproject/networks/lib/index.js
+++ b/node_modules/@ethersproject/networks/lib/index.js
@@ -170,6 +170,10 @@ var networks = {
     "arbitrum-goerli": { chainId: 421613, name: "arbitrum-goerli" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3" },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@ethersproject/networks/src.ts/index.ts b/node_modules/@ethersproject/networks/src.ts/index.ts
index daf97b7d..ed8dde80 100644
--- a/node_modules/@ethersproject/networks/src.ts/index.ts
+++ b/node_modules/@ethersproject/networks/src.ts/index.ts
@@ -208,6 +208,12 @@ const networks: { [name: string]: Network } = {

     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+
+    syscoin: { chainId: 57, name: "syscoin" },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum" },
+
+    rollux: { chainId: 570, name: "rollux" },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0xD5DD38942FE0a0913A35DeD04A702ba4951605A3" },
 }

 /**
\ No newline at end of file
