diff --git a/node_modules/@ethersproject/networks/lib.esm/index.js b/node_modules/@ethersproject/networks/lib.esm/index.js
index 918c9fb3..73654554 100644
--- a/node_modules/@ethersproject/networks/lib.esm/index.js
+++ b/node_modules/@ethersproject/networks/lib.esm/index.js
@@ -92,6 +92,18 @@
     };
     return func;
 }
+function crossChainProvider(url, network) {
+    const func = function (providers, options) {
+        if (providers.JsonRpcProvider) {
+            return new providers.JsonRpcProvider(url);
+        }
+        return null;
+    };
+    func.renetwork = function (network) {
+        return crossChainProvider(url, network);
+    };
+    return func;
+}
 const homestead = {
     chainId: 1,
     ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
@@ -173,6 +185,10 @@
     "arbitrum-goerli": { chainId: 421613, name: "arbitrum-goerli" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    rollux: { chainId: 570, name: "rollux", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@ethersproject/networks/lib/index.js b/node_modules/@ethersproject/networks/lib/index.js
index c0f624b0..dfafcb40 100644
--- a/node_modules/@ethersproject/networks/lib/index.js
+++ b/node_modules/@ethersproject/networks/lib/index.js
@@ -94,6 +94,18 @@
     };
     return func;
 }
+function crossChainProvider(url, network) {
+    var func = function (providers, options) {
+        if (providers.JsonRpcProvider) {
+            return new providers.JsonRpcProvider(url);
+        }
+        return null;
+    };
+    func.renetwork = function (network) {
+        return crossChainProvider(url, network);
+    };
+    return func;
+}
 var homestead = {
     chainId: 1,
     ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
@@ -175,6 +187,10 @@
     "arbitrum-goerli": { chainId: 421613, name: "arbitrum-goerli" },
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+    syscoin: { chainId: 57, name: "syscoin", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    rollux: { chainId: 570, name: "rollux", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
 };
 /**
  *  getNetwork
diff --git a/node_modules/@ethersproject/networks/src.ts/index.ts b/node_modules/@ethersproject/networks/src.ts/index.ts
index daf97b7d..ed8dde80 100644
--- a/node_modules/@ethersproject/networks/src.ts/index.ts
+++ b/node_modules/@ethersproject/networks/src.ts/index.ts
@@ -112,6 +112,22 @@
     return func;
 }

+function crossChainProvider(url: string, network: string | Network): Renetworkable {
+    const func = function(providers: any, options?: any): any {
+        if (providers.JsonRpcProvider) {
+            return new providers.JsonRpcProvider(url);
+        }
+
+        return null;
+    };
+
+    func.renetwork = function(network: Network) {
+        return crossChainProvider(url);
+    };
+
+    return func;
+}
+
 const homestead: Network = {
     chainId: 1,
     ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
@@ -214,6 +230,12 @@
 
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+
+    syscoin: { chainId: 57, name: "syscoin", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "syscoin-tanenbaum": { chainId: 5700, name: "syscoin-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+
+    rollux: { chainId: 570, name: "rollux", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
+    "rollux-tanenbaum": { chainId: 57000, name: "rollux-tanenbaum", ensAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", _defaultProvider: crossChainProvider("https://eth.llamarpc.com", "any") },
 }

 /**
\ No newline at end of file
